index.html

```diff
@@ -97,5 +97,208 @@ <h2 class="more_det">LET’S CONNECT</h2>
         </li>
       </ul>
     </section>
+    <section class="work-card">
```

Comment:
- [ ] Good job implementing the CSS grid system! However, the `display: grid` should be just the main sections wrappers.  The content itself should be aligned with `display: flex`.  

![image](https://user-images.githubusercontent.com/57421823/148490365-1efb51bd-1007-46c4-9c6e-a34d7aae2e4e.png)

That way, the project requirements will be met, and you will master both technologies 💪